### PR TITLE
[Forwardport] Removed space before ending sentence.

### PR DESCRIPTION
**Pull request by gelanivishal**
### Original Pull Request
https://github.com/magento/magento2/pull/16717

### Description

**Removed space before ending of statement and semicolon.**

### Fixed Issues (if relevant)

N/A

### Manual testing scenarios

N/A

### Contribution checklist
 - [X] Pull request has a meaningful description of its purpose
 - [X] All commits are accompanied by meaningful commit messages
 - [X] All new or changed code is covered with unit/integration tests (if applicable)
 - [X] All automated tests passed successfully (all builds on Travis CI are green)

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
Hi @gelanivishal. Thank you for your contribution 
Here is some useful tips how you can test your changes using Magento test environment.
Add the comment under your pull request to deploy test or vanilla Magento instance:
- `@magento-engcom-team give me test instance` - deploy test instance based on PR changes
- `@magento-engcom-team give me {$VERSION} instance` - deploy vanilla Magento instance

For more details, please, review the [Magento Contributor Assistant](https://github.com/magento/magento2/wiki/Magento-Contributor-Assistant) documentation

File: dev/tests/api-functional/testsuite/Magento/Quote/Api/CouponManagementTest.php [hunk 1]
```diff
@@ -297,4 +297,46 @@ public function testSetMyCouponSuccess()
 
         $this->assertEquals($quoteWithCoupon->getCouponCode(), $couponCode);
     }
+
+    /**
+     * @magentoApiDataFixture Magento/Sales/_files/quote.php
+     * @magentoApiDataFixture Magento/Checkout/_files/discount_10percent.php
+     */
+    public function testSetCouponWihSpaces()
```
Comment (sidolov) [inline dev/tests/api-functional/testsuite/Magento/Quote/Api/CouponManagementTest.php]: Looks like this changes not related to spaces removing. Please, revert.

**Review by sidolov — CHANGES_REQUESTED** [review]

File: dev/tests/api-functional/testsuite/Magento/Quote/Api/CouponManagementTest.php (hunk 1 above)
Comment (gelanivishal) [reply dev/tests/api-functional/testsuite/Magento/Quote/Api/CouponManagementTest.php]: Reverted.

File: app/code/Magento/Version/Controller/Index/Index.php [hunk 2]
```diff
@@ -38,12 +38,9 @@ public function __construct(Context $context, ProductMetadataInterface $productM
      */
     public function execute()
     {
-        $version = $this->productMetadata->getVersion();
```
Comment (sidolov) [inline app/code/Magento/Version/Controller/Index/Index.php]: Looks like this changes not related to this PR

**Review by sidolov — CHANGES_REQUESTED** [review]

**Comment by gelanivishal** [comment]
@sidolov Please check now.

File: app/code/Magento/Version/Controller/Index/Index.php [hunk 3]
```diff
@@ -45,6 +45,10 @@ public function execute()
         ) {
             return;
         }
+            
+        if (!isset($versionParts[0]) || !isset($versionParts[1])) {
+            return; // Major and minor version are not set - return empty response
+        }
```
Comment (sidolov) [inline app/code/Magento/Version/Controller/Index/Index.php]: This changes not related to PR, please, revert

**Review by sidolov — CHANGES_REQUESTED** [review]

**Comment by sidolov** [comment]
Hi @gelanivishal , please, take a look at failed tests.

**Comment by sidolov** [comment]
Hi @gelanivishal , I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Pull request closed without merging**